### ISSUE TITLE
Request to work with Iconize

### ISSUE THREAD

**Issue by gvivster** [scottTomaszewski/obsidian-inline-admonitions#14]
Hi there! It would help me to be able to use [Iconize](https://github.com/FlorianWoelki/obsidian-iconize) icons inside admonitions -- or, even better, to be able to specify an icon to appear at the front of my notes similarly to [Badges](https://github.com/gapmiss/badges). Would that be at all possible?

**Comment by scottTomaszewski** [comment]
Hi!

Can I get some clarity into what youre looking for?  From what I can tell, you can add iconize icons inside the inline-admonition content

![image](https://github.com/user-attachments/assets/88b22415-2bb2-4e6c-915c-45f49abe5e8d)

Are you looking to have an icon to be added automatically to inline-admonitions of a certain type?  For example, every time you make a `prefix` inline admonition that starts with `warn` as the trigger string, you want the admonition to have an alert icon?

Markdown: `warn dont forget to buy milk`
Expected output:
![image](https://github.com/user-attachments/assets/00eecf1c-0134-4a31-aac1-63d87fcab980)

**Comment by gvivster** [comment]
>Are you looking to have an icon to be added automatically to inline-admonitions of a certain type? For example, every time you make a prefix inline admonition that starts with warn as the trigger string, you want the admonition to have an alert icon?

Yes, exactly! And (it probably goes without saying) I'd like to be able to set which icon for each type, using the natively supported Lucide icons. (And if possible, I'd like to be able to search them to add them, like in Iconize - it would take a pain point out for me.)

**Comment by scottTomaszewski** [comment]
Release 1.5.2 is publicly available and adds support for Prefix Icons and Suffix Icons!

The plugin supports the icons that are known and available to obsidian, so it may not support iconize-specific icons, but the normal lucide icons are all available. If this limitation doesn't satisfy the request, please reopen this ticket
